lib/src/Extensions/src/openclext.cpp

```diff
@@ -64,11 +64,6 @@
 
 #include <vector>
 
-static inline cl_platform_id _get_platform(cl_platform_id platform)
-{
-    return platform;
-}
-
```

Comment:
Since this is a generated file, if these changes are needed they should be made to the template.  Let me know if you need help with this.

Reply:
I've removed it from the `openclext.cpp.mako`, I hope that that is the right file.

For additional context: this change was prompted by a warning fix.